README.md

```diff
@@ -1,17 +0,0 @@
-# seed-message-sender
-Seed Message Sender and Message Store
-
-## Apps & Models:
-  * message_sender
-    * Outbound
-    * Inbound
-
-## Metrics
-##### inbounds.created.sum
-`sum` Total number of inbounds created
-
-##### vumimessage.tries.sum
-`sum` Total number of outbound message send tries
-
-##### vumimessage.maxretries.sum
-`sum` Total number of outbound message send attempts that reached max retries
```

Comment:
It isn't up to date and there is a ticket to create formal documentation for all the services that haven't already gotten some.